lib/UnitFactory.js

```diff
@@ -15,6 +16,7 @@ UnitFactory.createUnit = function(unitType, location) {
                 "name": "gnat",
                 "description": "A swarm of annoying insects",
                 "hint": "Kill them with magic!",
+                "team": Team.B,
```

Comment:
should we make a team CREEP or something?

I was envisioning A/B etc.. be used for players.